src/lib/planviz/dot_config.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace opossum {
+
+enum class DotColor { Black, White, Transparent };
+
+enum class DotRenderFormat { PNG, SVG };
+
+struct DotConfig {
+  DotColor background_color = DotColor::Transparent;
+  DotRenderFormat render_format = DotRenderFormat::PNG;
```

Comment:
Why is the default here PNG, but the filename above was changed to SVG.
Also, I fail to understand the purpose of `dot_config.hpp`.

Reply:
fixed the `-Tsvg` thing.
As to the purpose of this file: I'm pretty sure the config will grow larger (things I can think of right now: print output column names on/off, statistics on/off, foreground color, ...) so I think it deserves to be its own class (struct) in its own file.